app/src/main/res/values/strings.xml

```diff
@@ -423,6 +423,7 @@
     </plurals>
     <string name="label_system_message_blocked_user">%1$s blocked this user</string>
     <string name="label_system_message_renamed_the_conversation">%1$s renamed the conversation\n%2$s</string>
+    <string name="label_system_message_team_member_left">%1$s was removed from the team</string>
```

Comment:
left or removed ? it two different actions
removed: removed by admin
left: leave the conversation by him self

Reply:
We don't have information about it in event data